modelator/src/event.rs

```diff
@@ -288,12 +288,14 @@ impl<'a, System> Runner<'a, System> {
         &mut self,
         system: &mut System,
         stream: &mut dyn Iterator<Item = Event>,
```

Comment:
This should also work without the need for a trait object (`dyn Trait`):

```suggestion
        stream: impl Iterator<Item = Event>,
```